libs/wingsdk/src/target-tf-azure/bucket.inflight.ts

```diff
@@ -264,16 +264,23 @@ export class BucketClient implements IBucketClient {
   }
 
   /**
-   * Copy object within the container
+   * Copy object within the bucket
    *
    * @param srcKey The key of the source object you wish to copy.
    * @param dstKey The key of the destination object after copying.
    * @throws if `srcKey` object doesn't exist.
    */
   public async copy(srcKey: string, dstKey: string): Promise<void> {
-    return Promise.reject(
-      `copy is not implemented: (srcKey=${srcKey}, dstKey=${dstKey})`
-    );
+    const srcBlobUrl = this.containerClient.getBlobClient(srcKey).url;
+    const dstBlobClient = this.containerClient.getBlockBlobClient(dstKey);
```

Comment:
what is the difference between `getBlobClient` to `getBlockBlobClient`?

Reply:
`BlobClient` is a generic client that works for all [blob types](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#blobs), so it's generally used for read operations where you don't know the type of the blob you are accessing.

`BlockBlobClient` is a specialized client that works only for block blobs. This means that you must be sure that the blob on which you are operating is of the correct type. As you can imagine, type-specific clients have a wider set of methods for manipulating a particular blob type.

Reply:
and why are we sure that the dest key is a block blob type?

Reply:
@tsuf239 Because you create the destination blob. You don't know what you are gonna read, but you definitely know what you want to write.

Reply:
@tsuf239 Because you are the one creating the destination blob. Basically, don't know what you are gonna read, but you definitely know what you want to write.

Reply:
@tsuf239 Because you are the one creating the destination blob. Basically, you don't know what you are gonna read, but you definitely know what you want to write.